### ISSUE TITLE
Teste de forward pass do modelo

### ISSUE THREAD

**Issue by ivano-lab** [ivano-lab/mini-gpt-zero#6]
Testar o modelo com um batch para garantir que o forward pass retorna os logits e o loss corretamente.
